### PR TITLE
release 4.0.0 release notes

### PR DESCRIPTION
**Pull request by joejulian**
**What type of PR is this?**

documentation

**What this PR does/ why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
none
```

**Checklist**

* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Review by dkoshkin — COMMENTED** [review]
Why are the release-notes changing so much from the previous RC?

**Comment by joejulian** [comment]
> Why are the release-notes changing so much from the previous RC?

Because I fixed the release-notes tool.

**Review by dkoshkin — APPROVED** [review]

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,15 +1,43 @@
 # Release Notes
 
-## testing-1.20-4.0.0-rc.4, v4.0.0-rc.4
+## stable-1.20-4.0.0, v4.0.0
+
+### ambassador
+  - bump ambasador to 1.12
```
Comment (samvantran) [inline RELEASE_NOTES.md:6]:
```suggestion
  - bump ambassador to 1.12
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -20,57 +48,116 @@
...
+  - feat(kommander-ui): allow workspace namespace to be configurable
+  #930 (@d2iq-dispatch)
 
-  - fix: update UI dependencies to mitigate CVE-2021-23337
-  #1075 (@mesosphere-mergebot)
+  - Update opsportal to mitigate CVE-2020-1971
+  #1107 (@mesosphere-mergebot)
 
   - fix: updating GitOps Source in Kommander.
   - fest: Show certification badges in Kommander project catalog.
```
Comment (samvantran) [inline RELEASE_NOTES.md:90]:
```suggestion
  - feat: Show certification badges in Kommander project catalog.
```

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -20,57 +48,116 @@
...
+  #976 (@d2iq-dispatch)
+
+  - UI - resolve kubecost performance issue
+  - UI - handle license loading state, show loading instead of invalid when license is missing a status
+  #1089 (@mesosphere-mergebot)
+
   - fix: updating GitOps Source in Kommander.
   - fest: Show certification badges in Kommander project catalog.
   - feat: Allow creating Root CA secret when attaching a firewalled cluster in Kommander.
   - feat: give list of loadbalncers when attaching a firewalled cluster in Kommander.
```
Comment (samvantran) [inline RELEASE_NOTES.md:125]: this looks duplicated on L108-112?

**Review by samvantran — APPROVED** [review]
Approving ahead of time, minor comments.

File: RELEASE_NOTES.md (hunk 1 above)
Comment (joejulian) [reply RELEASE_NOTES.md:6]: It's also a completely valueless note and should be removed.

File: RELEASE_NOTES.md (hunk 3 above)
Comment (joejulian) [reply RELEASE_NOTES.md:125]: Yep. The PR to master had the release/4.0 milestone set.

**Pull request merged**